### DISCUSSION TITLE
Questions about CPU Boost in Windows 11

### DISCUSSION THREAD

**Discussion by manciuszz** [seerge/g-helper#1180] [Q&A]
First things first, I'm running Windows 11 23H2 25931.1010.

I observed different behaviour on how 'CPU Boost' or more officially [PERFBOOSTMODE](https://learn.microsoft.com/en-us/windows-hardware/customize/power-settings/options-for-perf-state-engine-perfboostmode). For example, in Windows 10 when boost mode is set to 'Disabled', this literally disables turbo boosting, while other modes do make a noticeable difference. However, in Windows 11, it seems like nothing is happening? Observing ThrottleStop and changing the modes, you can see the CPU do something different. For example, setting boost mode to 'Disabled' when CPU is on full load, you can see the CPU disabling turbo boost for like a split second and then going back to previous behaviour. 

The issue is not exclusive to just 25931 build, I've observed the same behaviour on any other W11 build.

Does Windows 11 not support PERFBOOSTMODE anymore? This page https://learn.microsoft.com/en-us/windows-hardware/customize/power-settings/options-for-perf-state-engine-perfboostmode#applies-to would also be correct then.

Anyways, what are other people experiencing? Especially when they use 'Disabled' mode, does it really disable turbo boost (as it should) just like in Windows 10 ?

**Comment by seerge** [comment]
@manciuszz I can imagine it also depends on specific CPU (and chipset drivers).

On my G14 with Ryzen 9 6900HS different performance boost modes do make a (very slight) difference. And ``Efficient Aggressive` always scores highest in cinebench for example.

I'm moving this to discussions as this is not an "Issue" with the app.

**Comment by IceStormNG** [comment]
The setting depends on the CPU whether it does something. AMD CPUs respond more fine grained to this, while newer Intel CPUs usually only reacts to Disabled or anything else which means enabled.
It also depends on whether it uses CPPC or HWP, which means how much control windows actually has over the CPU's clocking behavior or whether the CPU does what it wants.
